rocks/BUILD

```diff
@@ -55,14 +53,17 @@ native_java_libraries(
     mac_deps = [
         "@maven//:com_google_ortools_ortools_darwin",
         "@maven//:com_google_ortools_ortools_darwin_java",
+        "@maven//:org_rocksdb_rocksdbjni_dev_mac",
     ],
     linux_deps = [
         "@maven//:com_google_ortools_ortools_linux_x86_64",
-        "@maven//:com_google_ortools_ortools_linux_x86_64_java"
+        "@maven//:com_google_ortools_ortools_linux_x86_64_java",
+        "@maven//:org_rocksdb_rocksdbjni_dev_linux",
     ],
     windows_deps = [
         "@maven//:com_google_ortools_ortools_win32_x86_64",
-        "@maven//:com_google_ortools_ortools_win32_x86_64_java"
+        "@maven//:com_google_ortools_ortools_win32_x86_64_java",
+        "@maven//:org_rocksdb_rocksdbjni",
```

Comment:
We don't yet have RocksDB (Debug) for Windows. Therefore we substitute it with RocksDB (Prod) for the time being.